go.mod

```diff
@@ -5,7 +5,7 @@ go 1.19
 require (
 	github.com/Masterminds/semver/v3 v3.2.0
 	github.com/aws/aws-sdk-go-v2/config v1.15.5
-	github.com/compose-spec/compose-go v1.9.0
```

Comment:
Can we have this in a separate commit or maybe another PR if not required for new filenames change?

Reply:
The new compose go has some new requirements on load that caused test failures, so probably better to review those separately.

Reply:
#1753, builds on this one